z64.h

```diff
@@ -0,0 +1,306 @@
+#define z64_magic_bean_addr						0x8011A652
+#define z64_megaton_hammer_addr					0x8011A653
+#define z64_arrow_light_addr					0x8011A654
+#define z64_nayrus_love_addr					0x8011A655
+#define z64_bottle_1_opt_addr					0x8011A656
+#define z64_bottle_2_opt_addr					0x8011A657
+#define z64_bottle_3_opt_addr					0x8011A658
+#define z64_bottle_4_opt_addr					0x8011A659
+#define z64_trade_itm_adult_addr				0x8011A65A
+#define z64_trade_itm_child_addr				0x8011A65B
+#define z64_qty_deku_stick_addr					0x8011A65C
+#define z64_qty_deku_nut_addr					0x8011A65D
+#define z64_qty_bomb_addr						0x8011A65E
+#define z64_qty_arrow_addr						0x8011A65F
+#define z64_qty_deku_seed_addr					0x8011A662
+#define z64_qty_bombchu_addr					0x8011A664
+#define z64_qty_magic_bean_addr					0x8011A66A
+#define z64_equipment_1_addr					0x8011A66C
+#define z64_equipment_2_addr					0x8011A66D
+#define z64_upgrades_addr						0x8011A670
+#define z64_quest_itm_1_addr					0x8011A674
+#define z64_quest_itm_2_addr					0x8011A675
+#define z64_quest_itm_3_addr					0x8011A676
+#define z64_quest_itm_4_addr					0x8011A677
+#define z64_dungeon_itm_addr					0x8011A678
+#define z64_dungeon_key_addr					0x8011A68C
+#define z64_defense_hearts_addr					0x8011A69F
+#define z64_gs_tokens_addr						0x8011A6A1
+#define z64_reset_gs_addr						0x8011B46C
+#define z64_flag_swordless_addr					0x8011B503
+#define z64_flag_interface_addr					0x8011B92F
+#define z64_flag_nayrus_addr					0x8011B998
+#define z64_countdown_tmr_addr					0x8011B9A2
+#define z64_opt_lang_addr						0x8011B9D9
+#define z64_control_input_ptr_addr				0x8011D730
+#define z64_scene_num_addr						0x801C8544
+#define z64_flag_switch_addr					0x801CA1C8
+#define z64_cutscene_ptr_addr					0x801CA208
+#define z64_scene_age_addr						0x801DA288
+#define z64_flag_reload_scn_addr				0x801DA2B4
+#define z64_scene_entrance_addr					0x801DA2BA
+#define z64_actor_pos_addr						0x801DAA38
+#define z64_levitate_addr						0x801DAA90
+#define z64_collision_detect_addr				0x801DAADE
+#define z64_actor_rot_addr						0x801DB25E
+
+
+#endif
+
+#if Z64_VERSION == Z64_OOT12
+#define z64_controller_1   (*(z64_controller_t*) 0x801C8D70)
+#define z64_rupees         (*(uint16_t*)         0x8011ACB4)
+#define z64_link_pos       (*(z64_xyz_t*)        0x801DB2F8)
+#define z64_link_rot       (*(z64_rot_t*)        0x801DB3A4)
+
+#define SetTextRGBA        ((SetTextRGBAProc)    0x800CC698)
+#define SetTextXY          ((SetTextXYProc)      0x800CC708)
+#define SetTextString      ((SetTextStringProc)  0x800CCDC8)
+
+/*Addresses*/
+#define z64_minimap_disable_1_addr              0x8006D3B0
+#define z64_minimap_disable_2_addr              0x8006DB40
+#define z64_frame_update_func_addr              0x8009B60C
+#define z64_frame_update_call_addr              0x8009D1D8
+#define z64_frame_input_call_func_addr          0x800A1290
+#define z64_frame_input_call_call_addr          0x800A1D8C
+#define z64_active_timer_addr                   0x8011AC8C
```

Comment:
This variable holds the current time of day. The name should be changed to something more suitable.